src/timecode_test_parse.rs

```diff
@@ -1012,6 +1029,23 @@ mod test {
         Ok(())
     }
 
+    /// tests that bad drop frame values fail to parse
+    #[rstest]
+    #[case("00:09:00:01", rates::F29_97_DF)]
+    #[case("00:08:00:01", rates::F59_94_DF)]
+    #[case("00:08:00:02", rates::F59_94_DF)]
+    #[case("00:08:00:03", rates::F59_94_DF)]
```

Comment:
Can you add a few more cases here with non-zero hours, including one where hours is over 24. I find that's a value not many libs test for since it's technically illegal, yet I've seen plenty of real-world values with 25+ in the hours field.